### PR TITLE
Api comments for book getlist

### PR DESCRIPTION
**Pull request by mvn-caonguyen-dn**
## Description
implement api comments for book
## Reference Requirements
- Product Backlog ID: PBI-39
- Task ID: #39.5

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

- config/define.php

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

File: routes/api.php [hunk 1]
```diff
@@ -20,4 +20,5 @@
 Route::group(['namespace' => 'Api'], function(){
     Route::get('books/{book}', 'BookController@show');
     Route::get('books/{id}/reviews', 'PostController@reviews');
+	Route::get('comments', 'CommentController@comments');
```
Comment (at-nhannguyen2) [inline routes/api.php]: fix spaces nha

**Review by ghost — COMMENTED** [review]
pls check comment

**Review by ghost — COMMENTED** [review]
LGTM

**Comment by at-huydq** [comment]
LGTM. good job.

**Pull request merged**